rotkehlchen/chain/manager.py

```diff
@@ -924,7 +924,9 @@ def _add_protocol_balances(self) -> None:
             for address, pool_balances in adex_balances.items():
                 for pool_balance in pool_balances:
                     eth_balances[address].assets[A_ADX] += pool_balance.adx_balance
+                    self.totals.assets[A_ADX] += pool_balance.adx_balance
```

Comment:
@vnavascues this is something you forgot to add when taking adex balances into account. Keep that in mind for next
time when including balances from a protocol.

We got per account balances and then we got total balances and adding should happen for both.